src/components/Icon.svelte

```diff
@@ -0,0 +1,49 @@
+<script context="module" lang="ts">
+  import Brain from "@svicons/boxicons-regular/brain.svelte";
+  import Code from "@svicons/boxicons-regular/code-alt.svelte";
+  import Edit from "@svicons/boxicons-regular/edit.svelte";
+  import Question from "@svicons/boxicons-regular/question-mark.svelte";
+  import Search from "@svicons/boxicons-regular/search.svelte";
+  import Transfer from "@svicons/boxicons-regular/transfer-alt.svelte";
+  import Bulb from "@svicons/boxicons-solid/bulb.svelte";
+  import Heart from "@svicons/boxicons-solid/heart.svelte";
+  import Conversation from "@svicons/boxicons-solid/conversation.svelte";
+  import Calendar from "@svicons/boxicons-solid/calendar.svelte";
+  import Slideshow from "@svicons/boxicons-solid/slideshow.svelte";
+  import Devices from "@svicons/boxicons-solid/devices.svelte";
+  import Vector from "@svicons/boxicons-solid/vector.svelte";
+</script>
+
+<script lang="ts">
+  export let icon: string;
```

Comment:
make a typescript string union type for the type instead, and have the dict be `Record<IconType, [whatever the icon type is]>